lib/fs.js

```diff
@@ -40,7 +40,7 @@ const {
 } = constants;
 
 const { _extend } = require('util');
-const pathModule = require('path');
+const { _makeLong, toNamespacedPath, resolve } = require('path');
```

Comment:
note that moving to this approach does have an impact on the ability to monkeypatch these methods. That, by itself is not a problem, but I believe it does make this PR semver-major

Reply:
So you mean that we will handle this in the future ?

Reply:
No, switching this now is fine, it just means there is a non-zero risk of breaking some code so it has to be treated as a semver-major.